### PR TITLE
Update rpcuser.py

### PR DESCRIPTION
**Pull request by jimjimmiejames**
Update line 39

### CODE REVIEW

File: share/rpcuser/rpcuser.py
Comment (chromatic) [inline share/rpcuser/rpcuser.py:39]: On my machine, this file is `dogecoin.conf`. Can you confirm on yours?

Comment (jimjimmiejames) [reply share/rpcuser/rpcuser.py:39]: I see it as 'dogecoind.conf' in dogecoin/contrib/init/

Comment (patricklodder) [reply share/rpcuser/rpcuser.py:39]: It's a different config file. The correct file referenced to here is https://github.com/dogecoin/dogecoin/blob/master/contrib/debian/examples/dogecoin.conf

**Review by jimjimmiejames — COMMENTED** [review]
redirect to correct .conf file

**Comment by chromatic** [comment]
:+1: Looks good to me. I'd like to squash these two commits into one before merging. If we can do that through the web interface, great! If not, let me know if you want help walking through it @jimjimmiejames or if you want one of us to do it for you.

**Comment by chromatic** [comment]
Actually, I think GitHub allows squash and merge as a single action from the Merge button here. I'm willing to give that a try.

**Review by chromatic — DISMISSED** [review]
:+1: Thanks for submitting this!

**Review by chromatic — APPROVED** [review]
Good to go. Thank you, @jimjimmiejames! We'll merge this after 24 hours per our review policy.

**Pull request merged**
